doc/source/ray-core/actors/terminating-actors.rst

```diff
@@ -66,6 +66,45 @@ flag ``no_restart=False`` to ``ray.kill``.
 For :ref:`named and detached actors <actor-lifetimes>`, calling ``ray.kill`` on
 an actor handle will destroy the actor and allow the name to be reused.
 
+.. tip::
+    
+    From Ray 2.0 onwards, you could view the actor exit details with :ref:`Ray's State API <state-api-overview-ref>``  to query the actors information, or go to the :ref:`Dashboard <observability-getting-started>`` Actor page to see more details. (These features all require installing ray with `pip install "ray[default]"`). 
+
+.. code-block:: bash
+  # This API is only available when you download Ray via `pip install "ray[default]"`
+  ray list actors --detail
+
+.. code-block:: bash
+
+  # You should be able to view the death cause of an actor in `death_cause` as part of the actor state.
```

Comment:
Here as well. Instead of having this code comment, just say, "you can verify actors are dead via this API."